setuptools/py36compat.py

```diff
@@ -33,11 +47,16 @@ def parse_config_files(self, filenames=None):
         if DEBUG:
             self.announce("Distribution.parse_config_files():")
 
-        parser = ConfigParser(interpolation=None)
+        parser = ConfigParser()
```

Comment:
I know interpolation causes problems when a percent appears in the setup.cfg (such as for a password). I think there are tests to protect this behavior (perhaps only in CPython).

Reply:
It's the other way round, interpolation as always been enabled, see: https://github.com/pypa/setuptools/issues/889#issuecomment-318363389

This compatibility code was actually not used, that's why I added a test first to make sure this behavior (interpolation enabled) did not change.

Reply:
Oh, right. Good point. I wonder then if that means we (I) need to update the code in distutils for Python 3.7 to restore interpolation before it goes live.

Reply:
Okay, reviewing the [history on the CPython issue](https://bugs.python.org/issue20754), I see that there's not yet a patch in CPython that's affected by this change.